qa/common/gen_qa_model_repository

```diff
@@ -65,6 +65,7 @@ HOST_SRCDIR=/tmp/gen_srcdir
 HOST_DESTDIR=/tmp/$TRITON_VERSION/qa_model_repository
 HOST_VARDESTDIR=/tmp/$TRITON_VERSION/qa_variable_model_repository
 HOST_IDENTITYDESTDIR=/tmp/$TRITON_VERSION/qa_identity_model_repository
+HOST_SIGDEFDESTDIR=/tmp/$TRITON_VERSION/qa_tf_tag_sigdef_repository 
```

Comment:
Once this PR is merged you should copy over the models to dlcluster. I'll let you know the directory

Reply:
I found the doc that has info about where to upload the data. Since this PR is going to be merged with the older release, I think a new version will be generated for that release as well.